src/main/java/newbank/server/Customer.java

```diff
@@ -3,9 +3,15 @@
 import java.util.ArrayList;
 
 public class Customer {
-	
+	private String username;
+	private String password;
 	private ArrayList<Account> accounts;
 	
+	public Customer( final String username, final String password ) {
+		this.username = username;
+		this.password = password;
+	}
+	
 	public Customer() {
```

Comment:
I don't think we should support this overloaded constructor anymore. Every customer shall have a username and password.